### ISSUE TITLE
./occ shows commands for disabled apps (user_ldap, files_encryption,...)

### ISSUE THREAD

**Issue by RealRancor** [owncloud/core#14836]
### Steps to reproduce
1. Have a running OC 8.0.2 (user_ldap was never enabled here)
2. Execute ./occ
3. Have a look at the help output 

### Expected behaviour
1. The help output could be stripped down. Maybe only ldap:show-remnants could be shown?

### Actual behaviour
1. The help output shows 8 available ldap commands
2. The help output shows encryption:migrate-keys command

### Server configuration
**ownCloud version:** 8.0.2

**Updated from an older ownCloud or fresh install:** Updated from 5 -> 6 -> 7 -> 8

**List of disabled apps:**
Disabled:
 - user_ldap
 - files_encryption

**Comment by RealRancor** [comment]
The same is valid for the files_encryption app. There:

```
encryption
  encryption:migrate-keys     migrate encryption keys
```

even if the app is not enabled.

**Comment by MorrisJobke** [comment]
Solved in 8.0.5 and 8.1.0

**Comment by RealRancor** [comment]
Ref: https://github.com/owncloud/core/pull/16747
